boot/zephyr/Kconfig

```diff
@@ -14,36 +14,21 @@ config MCUBOOT
 	select MPU_ALLOW_FLASH_WRITE if ARM_MPU
 	select USE_CODE_PARTITION if HAS_FLASH_LOAD_OFFSET
 
-# Define used by partition_manager.py to deduce size of partition
-config PM_PARTITION_SIZE_MCUBOOT
-	hex "Flash space reserved for bootloader."
-	default 0xc000
-	help
-	  Flash space set aside for the MCUBoot. Note, the name
-	  of this configuration needs to match the requirements set by the
-	  script 'partition_manager.py'. See pm.yaml.
+partition=MCUBOOT
+partition-size=0xc000
+source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_size"
 
+partition=MCUBOOT_SCRATCH
+partition-size=0x1e000
+source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_size"
 
-# Define used by partition_manager.py to deduce size of partition
-config PM_PARTITION_SIZE_MCUBOOT_SCRATCH
-	hex "Flash space reserved for scratch."
-	default 0x1e000
-	help
-	  Flash space set aside for the scratch area.
+partition=MCUBOOT_STORAGE
+partition-size=0x4000
+source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_size"
 
-# Define used by partition_manager.py to deduce size of partition
-config PM_PARTITION_SIZE_MCUBOOT_STORAGE
-	hex "Flash space reserved for storage."
-	default 0x4000
-	help
-	  Flash space set aside for the storage area.
-
-# Define used by partition_manager.py to deduce size of partition
-config PM_PARTITION_SIZE_MCUBOOT_PAD
-	hex "Flash space reserved for padding area."
-	default 0x200
-	help
-	  Flash space set aside for the padding area.
+partition=MCUBOOT_PAD
```

Comment:
Prompt `Flash space reserved for padding area.` became `Flash space reserved for MCUBOOT_PAD.`

Would be nice to have a comment that MCUBOOT_PAD is the padding area.

Reply:
The description of the MCUBOOT_PAD area is provided in the `pm.yml` which defines this partition.